### DISCUSSION TITLE
Glances Widget in Docker Container not working

### DISCUSSION THREAD

**Discussion by James-Firth** [Lissy93/dashy#1483] [Help and Support]
I'm running Dashy in a docker container and Glances on my host machine. When I load a glances widget it simply shows the default state (I know it's making requests as before I added the login info it gave a different error).

I see that Dashy makes an XHR request to `https://MYHOSTNAME/192.168.0.HOSTIP:61208/api/3/fs` (allcaps just show a redaction)

The response seems to be html + JS. I get a message about needing to enable JS when I look at the response preview so I presume it's a component getting passed back to the client.

But if I visit that address myself I get a 404 and if I visit `192.168.0.HOSTIP:61208/api/3/fs` myself I see a well formed JSON response.

Any ideas? Thanks in advance!

**Comment by lissy93** [comment]
Hmm, could you share exactly what you see in the browser console? [here's how](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#how-to-open-browser-console) 

If the URL it's making requests to is deffo correct, it sounds like either a CORS or auth issue. But I've not seen either of those on the Glances widget before.

**Reply by James-Firth** [reply]
Yup! Looks like I get an HTTP 404 error that _appears_ to be coming from dashy itself?

Attaching as a screenshot since I see some bearer tokens and don't want to copy those by accident and the `Uncaught (in promise)` object is huuuge so I can't easily inspect it.


![image psd(1)](https://github.com/Lissy93/dashy/assets/485809/6ec56784-8e02-47db-92b5-795acfe2b884)

<img width="306" alt="image" src="https://github.com/Lissy93/dashy/assets/485809/ff7f3258-ee43-475f-a65c-1e08a84884a2">


note when I hit the URl suffix (which I assume is what dashy reaches out to hit?) it's fine BUT it is HTTP not HTTPS if dashy enforces https requests

<img width="306" alt="image" src="https://github.com/Lissy93/dashy/assets/485809/2b415950-2ced-4811-80a7-52379bf04eb4">

EDIT: And yup the local domain name I have points to that IP so it _seems_ like it should be working
